### PR TITLE
[ISSUE-567] Release workflow

### PR DESCRIPTION
**Pull request by misanthropicat**
### Issue #567 

Created manually (or by GitHub API) triggered workflow for charts releasing

## PR checklist
- [x] Add link to the issue
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
[successful release workflow run on fork](https://github.com/misanthropicat/csi-baremetal/actions/runs/1358541603) triggered [this workflow in operator repo](https://github.com/misanthropicat/csi-baremetal-operator/actions/runs/1358542842). [Release in operator repo](https://github.com/misanthropicat/csi-baremetal-operator/releases/tag/v0.5.0) (no changes in changelog due to no pull requests in fork repo, but in the action log correct tags appeared, so it should work in base repo properly)
image

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+      
+      - name: Make version
+        id: vars
+        run: |
+          echo "::set-output name=version::$(make version)"
+
+      - name: Create asset
+        run: |
+          PACKAGE_NAME=csi-baremetal-deployment-${{ github.event.inputs.csi_operator_version }}.tgz
```
Comment (mishoyama) [inline .github/workflows/release.yml:33]: Where is the second asset?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+          DIR=./csi-baremetal-deployment-${{ steps.vars.outputs.version }}
+          cp -r ./charts $DIR
+          tar -czvf ./$PACKAGE_NAME $DIR
```
Comment (mishoyama) [inline .github/workflows/release.yml:36]: To make package you should use helm command

File: .github/workflows/release.yml (hunk 1 above)
Comment (misanthropicat) [reply .github/workflows/release.yml:33]: It is gathered in operator repo by release.yml workflow which is currently in progress.

File: .github/workflows/release.yml (hunk 2 above)
Comment (misanthropicat) [reply .github/workflows/release.yml:36]: Ok, and what about version? Do I need to use only operator version for `helm package --version=${{ github.event.inputs.csi_operator_version }}`?

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+jobs:
+  build:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Create issue
```
Comment (mishoyama) [inline .github/workflows/release.yml:16]: I think we can avoid issue creation

**Review by rkiselenko — APPROVED** [review]
LGTM

**Pull request merged**